Druid/VMMaker.extension.st

```diff
@@ -0,0 +1,8 @@
+Extension { #name : #VMMaker }
+
+{ #category : #'*Druid' }
+VMMaker >> stopOnErrors [
+
```

Comment:
Same here

Reply:
I don't know about this

#stopOnErrors in PharoVMMaker and SlangTyper for some reason don't provide a default?

I will check again deleting these methods, and evaluate with the new compilation we did today

Reply:
I moved this PR to the pharo-vm